src/utils/VerifyToken.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require("jsonwebtoken");
```

Comment:
I like how you've done this. Gonna use it as a reference for any future projects